### PR TITLE
#5562 - Create Circle Buttons

### PR DESCRIPTION
**Pull request by Miaplacidus**
This PR completes work detailed in issue #5562, which asks for the creation of a circular button containing only an icon (in primary and secondary variants). See design [here](https://www.figma.com/file/eIVp33a1oCu0AtcLwSbGjr/%F0%9F%9A%A7-Components-and-Patterns?type=design&node-id=3122-2051&mode=design&t=EEV66MAzwmhPjA9A-0)

## Link to Issue
Closes: #5562 

## Description of Changes
- Creates a new component called `CWCircleButton`
- Adds component to the component showcase page

## Test Plan
- visit `/components`
- navigate to section of page titled 'Circle Buttons' and ensure that they match look and behavior described in Figma above

### CODE REVIEW

**Review by Israellund — APPROVED** [review]
LGTM

**Review by CowMuon — APPROVED** [review]

File: packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWCircleButton/CWCircleButton.scss [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    }
+  }
+
+
+
+  .CircleButton {
+    line-height: 0;
+    width: fit-content;
+    border-radius: 50%;
+    padding: 6px;
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWCircleButton/CWCircleButton.scss:34]: it should have `cursor: pointer`

File: packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWCircleButton/CWCircleButton.scss [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+        border: none;
+      }
+    }
+  }
+
+
+
+  .CircleButton {
+    line-height: 0;
+    width: fit-content;
```
Comment (masvelio) [inline packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWCircleButton/CWCircleButton.scss:32]: I think the buttons are smaller versus designs. They are 28px whereas on the design they have 32px 
image


image

Comment (Miaplacidus) [reply packages/commonwealth/client/scripts/views/components/component_kit/new_designs/CWCircleButton/CWCircleButton.scss:32]: @masvelio 
Nice catch! Fixed!

**Pull request merged**